R/utils_render_common.R

```diff
@@ -288,7 +288,9 @@ perform_col_merge <- function(data,
         )
     }
 
-    if (type == "merge_range") {
+    if (type %in% c("merge_range", "merge_uncert")) {
+
+      data_tbl <- dt_data_get(data = data)
```

Comment:
The repeated `tidy_gsub` below is a bit problematic, because one `tidy_gsub` operation may return substituted text that is matched on a following `tidy_gsub`; when intuitively it should all be done in "one pass". glue (as used above) doesn't have this problem, but does have the problem of arbitrary code execution.

Ideally we would have one utility function that does this kind of template expansion.

Reply:
Now in a better place with use of our `glue_gt()` function:

```
glue_gt <- function(.x, ...) {
  glue::glue_data(.x, ..., .transformer = get, .envir = emptyenv())
}
```

There was a `glue_char()` function that just did `as.character()` after `glue::glue()`. It's no longer being used, so, it was removed.